src/app/pages/main/store/epics/index.ts

```diff
@@ -0,0 +1,25 @@
+import { RootStore, Dependencies } from '@core/store';
+import { ItemsList } from '@core/pages/main/namespace';
+import { of } from 'rxjs';
+import { map, catchError, switchMap } from 'rxjs/operators';
+import { ofType, Epic } from 'redux-observable';
+
+import { MainActionTypes, loadItemsFailed, loadItemsSuccess } from '../actions';
+
+const executeGetItemsEpic: Epic<any, any, RootStore, Dependencies> = (
+  action$,
+  _state$,
+  { itemsService: { getItems } },
+) => {
+  return action$.pipe(
+    ofType(MainActionTypes.LOAD_ITEMS),
+    switchMap(({ payload }) => {
+      return getItems(payload).pipe(
```

Comment:
getItems should be simple Promise based service, and you can here make from Promise based sercie Observale and use pipe on it.